sdccc/src/main/java/com/draeger/medical/sdccc/messages/MessageStorage.java

```diff
@@ -131,6 +131,9 @@ public class MessageStorage implements AutoCloseable {
             "getInboundMessagesByBodyType called on closed storage";
     private static final String GET_INBOUND_MESSAGE_BY_TIME_INTERVAL_CALLED_ON_CLOSED_STORAGE =
             "getInboundMessagesByTimeInterval called on closed storage";
+    // getInboundMessagesByTimestampAndBodyType
```

Comment:
What is this comment good for? IMHO, IDEA does a great job in visualizing where a constant is used...

Reply:
That comment did not belong there, removed it.